etc/benchmarks/convert_to_csv.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# This script is meant to be used on the output of benchmark runs to generate a csv file that can be
+# more easily ingested in google sheets or your spreadsheet/data anaylsis tool of choice
+# note that you will also see the output of the csv file printed in the terminal
+usage=$(/bin/cat <<EOM 
```

Comment:
```suggestion
USAGE=$(/bin/cat <<EOM 
```

Since this is a constant, the general convention is to use all caps.